temporalcli/commands.schedule_test.go

```diff
@@ -249,6 +262,69 @@ func (s *SharedServerSuite) TestSchedule_List() {
 		ok = ok || j.ScheduleId == schedId
 	}
 	s.True(ok, "schedule not found in jsonl result")
+
+	// JSON query (match)
+
+	res = s.Execute(
```

Comment:
I think you should probably use `Eventually` here due to eventual consistency of the visibility store.

Reply:
I rearranged the test assertions to ensure eventually visibility for the first case.